src/main/java/org/apache/commons/compress/harmony/pack200/IntList.java

```diff
@@ -120,6 +120,19 @@ public int get(final int location) {
         throw new IndexOutOfBoundsException("" + location);
     }
 
+
+
+    private int calculateIncrement(final int size, final int required) {
+        int increment = size / 2;
+        if (required > increment) {
+            increment = required;
+        }
+        if (increment < 12) {
+            increment = 12;
+        }
+        return increment;
```

Comment:
You can use `NumberUtils` for this:
```suggestion
        return NumberUtils.max(size / 2, required, 12);
```

Additionally I would:

- Make the method `static`.
- Optionally extract the constant 12 to a `MIN_INCREMENT` constant, although for this single usage, it is not really necessary.

Reply:
yeah 
the use of NumberUtils would be more clear for this case